src/dto/Validator.java

```diff
@@ -0,0 +1,66 @@
+package dto;
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+            public List<String> validate(Object object) throws IllegalAccessException {
+                List<String> arraylist = new ArrayList<>();
+                Class<?> objectClass = object.getClass();
+                Field[] completedFields = objectClass.getDeclaredFields();
+
+                for (Field declaredField : completedFields) {
+                    declaredField.setAccessible(true);
+                    Object fieldValue = declaredField.get(object);
+
+                    if (declaredField.isAnnotationPresent(Length.class) && fieldValue != null) {
+                        Length length = declaredField.getAnnotation(Length.class);
+                        if (fieldValue.toString().length() < length.min() || fieldValue.toString().length() > length.max()) {
```

Comment:
On lines -23, 33, 42, 50, 58 you have a lot of duplicated string literals. 
Duplicated string literals make the process of refactoring error-prone, since you must be sure to update all occurrences. On the other hand, constants can be referenced from many places, but only need to be updated in a single place. Define a constant instead.